pkg/test/runner/config.go

```diff
@@ -23,13 +23,23 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// ImperativeTestCaseConfig contains the config only for imperative
+// function run
+type ImperativeTestCaseConfig struct {
+	ExecPath string            `json:"execPath,omitempty" yaml:"execPath,omitempty"`
+	Image    string            `json:"image,omitempty" yaml:"image,omitempty"`
+	Args     map[string]string `json:"args,omitempty" yaml:"args,omitempty"`
```

Comment:
What's the plan for passing function config?

Reply:
Add the field for fn config